scroll_menu.py

```diff
@@ -7,45 +11,88 @@
     CompletionsMenu,
     Float,
     FloatContainer,
+    FormattedTextControl,
     HSplit,
     Layout,
     ScrollablePane,
     VSplit,
+    Window,
 )
-from prompt_toolkit.widgets import Frame, Label, TextArea
+from prompt_toolkit.widgets import Button, Frame, Label
+
+
+class ScrollableMenu:
+    """Main class for scrollable menu"""
```

Comment:
```suggestion
    """
    Scrollable Menu UI Element.
    
    Displays contents of the a given working directory and allows users to inspect files. 
    In the future users can drill down into directories.
    """
```
Does this description look good to you? You can commit it from the PR UI.